README.rst

```diff
@@ -3,8 +3,33 @@ adb-utils
 
 Utility and Service scripts for the `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_.
 
+adb-utils provides following content to ADB Vagrant box
+
+* OpenShift as a service
+
+  adb-utils uses OpenShift docker containers and provides systemd service unit file which inludes some setup scripts for the purpose. With the systemd unit file, OpenShift can be configured and run in ADB with just "systemctl start openshift" command.
+
+* SCCLI
+
+The idea behind providing SCCLI is to provide a way for users who just want to do vagrant ssh in to the ADB vagrant box and do their work. With this CLI they can setup required providers e.g. single node Kubernetes , OpenShift , Mesos marathan etc with single CLI commands.
```

Comment:
SCCLI provides a single point of entry for managing services inside of the ADB.  This is primarily of interest to users who use the ADB as a VM they access via `vagrant ssh`.

This CLI can clean the environment and ensure that the requested service is set up properly.